client/src/components/style.css

```diff
@@ -119,3 +108,123 @@ a:hover {
   box-shadow: 0 1px 0 rgba(255,255,255,.6) inset;
   white-space: nowrap;
 }
+
+.flex-gallery {
+  display: flex;
+  flex-wrap: wrap;
+  /* centre the content*/
+  max-width: 1024px;
+  margin: 0 auto;
+  background-color: #fff;
+}
+
+.box{
+  /* flex-grow and flex-shrink changes images proportionlly*/
+  flex-grow:1;
+  flex-shrink: 1;
+  flex-basis: auto;
+  /* flex-basis: calc(25%-10px); */
+  width: 128px;
+  height: 128px;
+  /* background-color: #ADB1B8;
+  color: #fff;
+  font-size: 48px; */
+  margin: 5px;
+  margin-top: 30px;
+  position: relative;
+  /* width: 70%; */
+  /* margin: 0 auto; */
+  /* height: auto;
+  max-height: 70vh; */
+  /* margin-top: calc(100vh - 85vh - 20px); */
+  /* background: #fff; */
+  /* border-radius: 4px;
+  padding: 20px;
+  border: 1px solid #999;  */
+  overflow: auto;
+  /* overflow: scroll; */
+  /* overflow: hidden; */
+
+}
+
+/* .imgWrapper{
+  overflow: hidden;
+  padding-bottom: 119.8%;
+  position: relative;
+
+} */
+
+img{
+  width: 100%;
+  /* position: absolute;
+  top: 50%;
+  transform: translate(-50%); */
+
+}
+
+.closeIcon {
+  content: 'x';
+  cursor: pointer;
+  width: 25px;
+  height: 25px;
+  text-align: center;
+  font-size: 20px;
+  color: -internal-light-dark(black, white);
+  background: #ededed;
+  border: #ededed;
+}
+
+.popover-modal {
+    position: absolute;
+    border-radius: 8px;
+    border: 1px solid;
+    border-color: #D5D9D9;
+    box-shadow: 0 0 14px 0 rgba(15,17,17,.5);
+    height: auto;
+    /* height: 100%; */
+    overflow: auto;
+    /* opacity: 1; */
+    background-color: #fff;
+    z-index:1;
+
+}
+
+.popover-header {
+  border-bottom: 1px solid #D5D9D9;
+    background-color: #F0F2F2;
+    border-radius: 8px 8px 0 0;
+    padding: 0 24px;
+
+  /* padding: 16px 0;
+  min-height: 56px;
+  margin-right: 28px;
+  line-height: 24px;
+  font-size: 16px;
+  background: linear-gradient(to bottom,#f7f7f7,#eaeaea); */
+}
+/*
+img {
+  border-radius: 50%;
+  width: 34px;
+} */
+
+.box1 {
+  position: relative;
+  width: 70%;
+  margin: 0 auto;
+  height: auto;
+  max-height: 70vh;
+  margin-top: calc(100vh - 85vh - 20px);
+  background: #fff;
+  border-radius: 4px;
+  padding: 20px;
+  border: 1px solid #999;
+  overflow: auto;
+}
+
+.profile-content {
+  display: table-cell;
+  vertical-align: middle;
+  min-height: 14px;
+
+}
```

Comment:
Consider using styled-component theming for more global CSS rules [https://styled-components.com/docs/advanced#theming](https://styled-components.com/docs/advanced#theming). Mixing a CSS style sheet with styled-component might result in unpredictable CSS behavior

Reply:
Thank you. Yes, I am planning to remove it. I started transitioning to styled components but still need to review what is in use from CSS and clean it up. I will look into for next PR.